tests/BUILD.bazel

```diff
@@ -25,3 +25,22 @@ build_test(
         "//python/entry_points:py_console_script_binary_bzl",
     ],
 )
+
+# Used for manual tests when downloading a toolchain for a particular platform
```

Comment:
Use and/or move these to test/support (there's a couple platform defs there already)